websocket tck bundle, documentation and signature test.

gurunrao opened the pull request:
websocket tck bundle, documentation and signature test.

scottmarlow commented:
Thanks @gurunrao!  

Fyi, @markt-asf can now be selected as a reviewer for projects that he leads.  He now has `triage` permissions for the Platform TCK as does @lukasj as well.  

Any Platform TCK committer can grant the `trigage` role to others (e.g. likely EE Spec project leads) as well as per https://www.eclipse.org/projects/handbook/#pmi-contributors

gurunrao commented:
> Thanks @gurunrao!
> 
> Fyi, @markt-asf can now be selected as a reviewer for projects that he leads. He now has `triage` permissions for the Platform TCK as does @lukasj as well.
> 
> Any Platform TCK committer can grant the `trigage` role to others (e.g. likely EE Spec project leads) as well as per https://www.eclipse.org/projects/handbook/#pmi-contributors

Thanks Scott. Adding `trigage` role helps. we need review from SPEC committers, who are not committers at Platform TCK . I have added Mark-asf as reviewer for the PR, will implement comments as part of new PR.

@markt-asf - Please feel free to add review comments, will open a new PR for addressing comments.
